docs_app/tools/decision-tree-generator/src/lib/interfaces.ts

```diff
@@ -57,19 +57,13 @@ export type ApiUnion =
   | 'mergeMap'
   | 'mergeMapTo'
   | 'mergeScan'
-  | 'multicast'
```

Comment:
I noticed `'concat'` two times in lines 24 and 25, can you remove one of them?

Reply:
Done. Technically, it doesn't matter to `git` which one I deleted. 😆